internal/store/indexes.go

```diff
@@ -18,20 +18,20 @@ import (
 	"fmt"
 
 	"github.com/mongodb/mongodb-atlas-cli/internal/config"
-	atlas "go.mongodb.org/atlas/mongodbatlas"
+	atlasv2 "go.mongodb.org/atlas/mongodbatlasv2"
 )
 
 //go:generate mockgen -destination=../mocks/mock_indexes.go -package=mocks github.com/mongodb/mongodb-atlas-cli/internal/store IndexCreator
 
 type IndexCreator interface {
-	CreateIndex(string, string, *atlas.IndexConfiguration) error
+	CreateIndex(string, string, *atlasv2.IndexRequest) error
 }
 
 // CreateIndex encapsulate the logic to manage different cloud providers.
-func (s *Store) CreateIndex(projectID, clusterName string, index *atlas.IndexConfiguration) error {
+func (s *Store) CreateIndex(projectID, clusterName string, index *atlasv2.IndexRequest) error {
 	switch s.service {
 	case config.CloudService, config.CloudGovService:
-		_, err := s.client.(*atlas.Client).Indexes.Create(s.ctx, projectID, clusterName, index)
+		_, err := s.clientv2.RollingIndexApi.CreateRollingIndex(s.ctx, projectID, clusterName).IndexRequest(*index).Execute()
```

Comment:
@wtrocki This is failing due to:
Error: https://cloud-dev.mongodb.com/api/atlas/v2/groups/6421987cdc060f20518edb62/clusters/Cluster0/index POST: HTTP 500 (Error code: "UNEXPECTED_ERROR") Detail: Unexpected error. Reason: Internal Server Error. Params: []

Command:bin/atlas clusters index create swtyagi_index --clusterName Cluster0  --collection listingsAndReviews --db sample_airbnb --key accommodates:number --projectId 6421987cdc060f20518edb62

Reply:
This is now addressed :-)